docs/sql-ref-syntax-qry-select-transform.md

```diff
@@ -0,0 +1,263 @@
+---
+layout: global
+title: TRANSFORM
+displayTitle: TRANSFORM
+license: |
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+ 
+     http://www.apache.org/licenses/LICENSE-2.0
+ 
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+---
+
+### Description
+
+The `TRANSFORM` clause is used to specifies a Hive-style transform (SELECT TRANSFORM/MAP/REDUCE)
+query specification to transform the input by running a specified script. Users can
+plug in their own custom mappers and reducers in the data stream by using features natively supported
+in the Spark SQL. e.g. in order to run a custom mapper script `map_script` and a custom
+reducer script `reduce_script` the user can issue the following command which uses the TRANSFORM
+clause to embed the mapper and the reducer scripts.
+
+In default format, columns will be transformed to `STRING` and delimited by tabs before feeding
+to the user script, Similarly, all `NULL` values will be converted to the literal string `\N` in order to
```

Comment:
For complete clarity that it is a string, _also_ put double quotes around the string like `"\N"`